src/services/ethereum/utils.test.ts

```diff
@@ -0,0 +1,17 @@
+import { getProvider } from './utils';
+import { ethers } from 'ethers';
+
+import { ARBITRUM_URI, ARBITRUM_RINKEBY_URI, ETHERUM_URI, ROPSTEN_URI } from '../../env';
+
+describe('getProvider', () => {
+  test('get RPC URL of the network based on chainID', () => {
+    // given / arrange
+    const chainID = 42161;
+
+    // when / act
+    const provider = getProvider(chainID);
+
+    // then / assert
+    expect(provider).toStrictEqual(new ethers.providers.JsonRpcProvider(ARBITRUM_URI));
```

Comment:
Not sure what you're testing here exactly. Are you checking if the returned object is a JsonRpcProvider instance? That might be ok, but then does that match the description in the `test(...)`?
Tests need to be clear and simple. Here are some suggestions for the test cases (you can use them for `test` descriptions:
- returns JsonRpcProvider instance
- returns proper instance for Arbitrum (you can test other supported chains too; if you do then you can put those tests in a loop)
- falls back to Ethereum for unknown chainId